docs/guides/chip_tool_guide.md

```diff
@@ -164,24 +168,28 @@ I: 278 [DL] Manufacturing Date: (not set)
 I: 281 [DL] Device Type: 65535 (0xFFFF)
 ```
 
-In this printout, the _discriminator_ is `3840 (0xF00)` and the _setup PIN code_
-is equal to `20202021`.
+In this printout, the discriminator is `3840 (0xF00)` and the setup PIN code is
+`20202021`.
 
 ### Step 6: Commission Matter device into an existing IP network
 
 Before communicating with the Matter device, first it must join an existing IP
 network.
 
-Matter devices may use different commissioning channel. Typically, devices which
-are not yet connected to the target IP network use Bluetooth LE as the
-commissioning channel. However, if the device has already joined an IP network,
-the only thing needed is to commission it to the Matter network over the IP
-protocol.
+Matter devices can use different commissioning channel:
+
+-   Devices that are not yet connected to the target IP network use Bluetooth LE
+    as the commissioning channel.
+-   Devices that have already joined an IP network only need to use the IP
+    protocol for commissioning to the Matter network.
 
 #### Commissioning over Bluetooth LE
 
-This section describes how your device can join the existing IP network over
-Bluetooth LE and then be commissioned into a Matter network.
+In this case, your device can join the existing IP network over Bluetooth LE and
+then be commissioned into a Matter network.
+
+Different scenarios are available for the Thread network and the Wi-Fi network,
+respectively.
```

Comment:
```suggestion
Different scenarios are available for Thread networks and Wi-Fi networks.
```
perhaps?

Reply:
This is meant to hint to the subsections that follow. Rephrased.